sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -494,6 +494,78 @@ export class StudentService extends RecordDataModelService<Student> {
     return searchQuery.getMany();
   }
 
+  /**
+   * Search students based on the search criteria.
+   * @param searchCriteria options to search by firstName,
+   * lastName, appNumber or sin.
+   * @returns list of students.
+   */
+  async searchStudentApplicationForInstitution(
+    institutionId: number,
+    searchCriteria: {
+      firstName?: string;
+      lastName?: string;
+      appNumber?: string;
+      sin?: string;
+    },
+  ): Promise<Student[]> {
+    const searchQuery = this.repo
+      .createQueryBuilder("student")
+      .select([
+        "student.id",
+        "student.birthDate",
+        "user.firstName",
+        "user.lastName",
+        "sinValidation.sin",
+      ])
+      .innerJoin(
+        Application,
+        "application",
+        "application.student.id = student.id",
+      )
+
+      .innerJoin("application.location", "pirLocation")
```

Comment:
Maybe it would be safe to use left join here since we still are not saving the PIR location right on submission.
Even if we start saving, I believe that it would be better to not rely on it using the INNER.